integration/tests/unowned_cluster/unowned_cluster_test.go

```diff
@@ -110,8 +110,8 @@ var _ = Describe("(Integration) [non-eksctl cluster & nodegroup support]", func(
 				"--nodegroup", ng1,
 				"--verbose", "2",
 			)
-			// It takes a second or 2 for the above set to take effect
-		Eventually(func() *gbytes.Buffer { return cmd.Run().Out }, time.Second*10).Should(gbytes.Say("key=value"))
+			// It sometimes takes more than 10 seconds for the above set to take effect
```

Comment:
Interesting, not important but this could also be written with some combination of `Eventually` and:

```
Expect(cmd).To(RunSuccessfullyWithOutputStringLines(
			ContainElement(ContainSubstring("key=value")),
		))
```

Reply:
Yeh I tried that initially but could not get it to cooperate, maybe I missed a trick 🤔 .
I may add it later, for now I just want to get these fixes in.